pkg/iss/iss.go

```diff
@@ -108,6 +116,9 @@ type ISS struct {
 	// These fields should only be set at initialization and remain static
 	// --------------------------------------------------------------------------------
 
+	// IDs of modules ISS interacts with.
+	moduleConfig ModuleConfig
```

Comment:
Maybe pass by reference for consistency?

Reply:
Fixed.